packages/@ember/controller/index.ts

```diff
@@ -58,40 +44,16 @@ const ControllerMixin = Mixin.create(ActionHandler, {
     as part of the application's initialization process. In most cases the
     `target` property will automatically be set to the logical consumer of
     actions for the controller.
-
-    @property target
-    @default null
```

Comment:
This was just a miss in the original writeup, so not your fault, but we need to keep these! They're part of how the doc generator knows what to output.

Reply:
Whoops! Re-added 😅